client/src/app/models/site.coffee

```diff
@@ -0,0 +1,10 @@
+Backbone = require('backbone')
+
+module.exports = Backbone.Model.extend(
+  urlRoot: 'http://localhost:8989/sites'
```

Comment:
Don't think this should have the FQDN should it?  Should probably be a relative URI